composer/workflows/airflow_db_cleanup.py

```diff
@@ -375,9 +360,11 @@ def cleanup_function(**context):
     except ProgrammingError as e:
         logging.error(e)
         logging.error(
-            str(airflow_db_model) + " is not present in the metadata."
+            str(airflow_db_model) + " is not present in the metadata. "
             "Skipping...")
 
+    session.close()
```

Comment:
There has to be wrapper (context manager) in Airflow to create/close session - it should close session even exception raised

Reply:
I move it to finally section to stay close to original DAG, since not all airflow support context manager.